RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Pinny Release Checklist
+
+Take this checklist and paste it into your release PR. Ensure steps are followed in order.
+
+- [ ] Tests pass in the terminal and in supported browsers.
+  - See [README](./README.md#browser-compatibility) for list of supported browsers.
```

Comment:
Looks great! One thought - should we add a list of UI bugs that we should monitor for regressions? The use case for this would be for visual bugs that aren't easily testable by automated tests.

For example: "On Android 4.x/Chrome, if you scroll while opening up Pinny, then close it, the background is in the wrong place."

I know we might not have any at the moment, but I'm proposing adding it to the checklist so that we keep track of them for the future.